ingest/annotations.py

```diff
@@ -94,7 +94,8 @@ def create_data_frame(self):
             # multiIndex = pd.MultiIndex.from_tuples(index)
         index = pd.MultiIndex.from_tuples(columns)
         self.file = self.open_file(
-            self.file_path, open_as='dataframe', dtype=dtypes, names=index, skiprows=2
+            self.file_path, open_as='dataframe', dtype=dtypes, names=index, skiprows=2,
+            keep_default_na=False,
```

Comment:
This change prevents the [issue seen by Jon](https://broadinstitute.slack.com/archives/CSM9QUHMF/p1593097722253300) in SCP-2545 where NA in a group-based annotation in a cluster file is converted to NaN by ingest (via Pandas) - for group-based annotation, the annotation should remain a string. 

Eno and I chatted and there are outstanding questions about whether points with NA values should be subsampled and also if we're depending on Pandas detection of empty cells, we should at least add na_values with the empty string and find out which of NaN, nan, etc might be outputted by Seurat or ScanPy that we should be treating as NA values.